README.md

```diff
@@ -9,492 +9,67 @@ Search Insights lets you report click, conversion and view metrics using the [Al
-
-### Sending events to multiple Algolia applications
-
-The Search Insights library sends all events to the Algolia application it has been initialized with, by default. In some cases, you might have data coming from more than one Algolia application which all need events.
-
-You can use a single instance of the Search Insights library while still customizing the Algolia credentials on a per-event basis. All click, view and conversion methods support an additional argument where you can specify credentials that will override those set during initialization (which is optional).
-
-```js
-aa('clickedObjectIDsAfterSearch', {
-  index: 'INDEX_NAME',
-  eventName: 'Click item',
-  queryID: getQueryID(),
-  objectIDs: ['object1'],
-  positions: [42],
-}, {
-  headers: {
-    'X-Algolia-Application-Id': 'OTHER_APP_ID',
-    'X-Algolia-API-Key': 'OTHER_SEARCH_API_KEY'
-  }
-});
-```
-
-This also works with event batching, using `sendEvents`:
-
-```js
-aa('sendEvents', [
-  {
-    eventType,
-    eventName,
-    userToken,
-    ...
-  }
-], {
-  headers: {
-    'X-Algolia-Application-Id': 'OTHER_APP_ID',
-    'X-Algolia-API-Key': 'OTHER_SEARCH_API_KEY'
-  }
-});
-```
-
-| Option    | Type                     | Description                                                  |
-| --------- | ------------------------ | ------------------------------------------------------------ |
-| `headers` | `Record<string, string>` | A dictionary of headers that will override those set by default. You can check out a list of headers for authentication in the [Algolia API reference](https://www.algolia.com/doc/rest-api/insights/#authentication). |
+For information on how to set up `search-insights.js` in a Node.js environment, see our guide on [Installing and Initializing the Insights Client for Node.js](./docs/nodejs.md).
 
-## Examples
+[insights-api]: https://www.algolia.com/doc/rest-api/insights/
+[insights-js-docs]: https://www.algolia.com/doc/api-client/methods/insights/#install-the-insights-client
+[instantsearch]: https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/
+[instantsearch-guide]: https://www.algolia.com/doc/guides/building-search-ui/events/js/
+[autocomplete]: https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete/
+[autocomplete-guide]: https://www.algolia.com/doc/ui-libraries/autocomplete/guides/sending-algolia-insights-events/
 
-The following examples assume that the Search Insights library is loaded.
+## Documentation
 
-- [InstantSearch.js](https://github.com/algolia/search-insights.js/blob/main/examples/INSTANTSEARCH.md)
-- [AlgoliaSearch Helper](https://github.com/algolia/search-insights.js/blob/main/examples/HELPER.md)
+Documentation for `search-insights.js` can be found in our main [Algolia Docs](https://algolia.com/docs) website.
```

Comment:
Is there a more specific link?

Reply:
The very next line is the more specific link

https://github.com/algolia/search-insights.js/pull/489/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R72

Reply:
> Insights API Client, but that's not "search-insights.js"
These unfortunately are the same thing, not sure the best course of action to reconcile this.